HW2/HW2.sql

```diff
@@ -0,0 +1,77 @@
+USE baseball;
+
+# You have probably already corrected this
+#ALTER TABLE inning MODIFY COLUMN game_id INT UNSIGNED NOT NULL;
+
+CREATE INDEX game_id ON batter_counts(game_id);
+
+#assignment
+#Calculate the batting average using SQL queries for every player
+CREATE TABLE batter_ba as
+SELECT a.batter,
+       CASE WHEN sum(a.atBat)>0 THEN
+       format(sum(a.Hit) / sum(a.atBat),3)
```

Comment:
Maybe use round here instead.  Not sure if format rounds the number or just cuts it off